pkg/configs/client/client.go

```diff
@@ -119,20 +120,20 @@ type dbStore struct {
 // GetRules implements ConfigClient.
 func (d dbStore) GetRules(since configs.ID) (map[string]configs.VersionedRulesConfig, error) {
```

Comment:
Can you add a context here too please?

Reply:
Yes, I wanted to wait til #1258 was merged so I could avoid more merge conflicts, but I'll add it here.